tests/neg/i18657.check

```diff
@@ -0,0 +1,17 @@
+-- [E086] Syntax Error: tests/neg/i18657.scala:2:27 --------------------------------------------------------------------
+2 |val f: (Int, Int) => Int = Integer.compare(_ + 1, _)  // error
+  |                           ^^^^^^^^^^^^^^^^^^^^^^^^^
+  |                           Wrong number of parameters, expected: 2
+  |---------------------------------------------------------------------------------------------------------------------
+  | Explanation (enabled by `-explain`)
+  |- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
+  | The function literal
+  |
+  |     _$2 => Integer.compare(_$1 => _$1 + 1, _$2)
+  |
+  | has 1 parameter. But the expected type
+  |
+  |     (Int, Int) => Int
+  |
+  | requires a function with 2 parameters.
```

Comment:
Maybe the reader can infer from looking at the printed tree, but perhaps also explain that placeholder parameters always infer parameters 1 level of nesting up from their position (however best to explain that).

Reply:
Indeed, not sure how to explain that, in a situation like this, where context is missing.